event_manager: Add event postprocessing hooks (#6805)
nrfconnect/sdk-nrf: doc/nrf/releases/release-notes-changelog.rst

* Added:

>   * Universal hooks for event postprocessing in a form event_manager_event_postprocess_N where N is in a range between 1 and 4.

Reviewer: ```suggestion
  * Universal hooks for event postprocessing in the form event_manager_event_postprocess_N, where N is between 1-4 values.
```
